content/en/observability_pipelines/destinations/amazon_opensearch.md

```diff
@@ -40,4 +53,6 @@ A batch of events is flushed when one of these parameters is met. See [event bat
 
 [1]: https://app.datadoghq.com/observability-pipelines
 [2]: /observability_pipelines/destinations/#event-batching
-[3]: /observability_pipelines/destinations/#template-syntax
\ No newline at end of file
+[3]: /observability_pipelines/destinations/#template-syntax
+[4]: https://docs.aws.amazon.com/opensearch-service/latest/developerguide/gsgupload-data.html
```

Comment:
@mikhacim can you confirm that this is the correct link for Amazon OpenSearch bulk API?